policyengine_uk/parameters/gov/hmrc/tax_free_childcare/contribution/rate.yaml

```diff
@@ -0,0 +1,12 @@
+description: Government contribution rate for tax-free childcare (£2 for every £8 spent)
+metadata:
+  unit: /1
+  period: year
+  label: Tax-free childcare contribution rate
```

Comment:
Let's use consistent capitalisation. Gov.uk capitalises Tax-Free Childcare so let's copy that. https://www.gov.uk/tax-free-childcare